.github/workflows/run-checker-daily.yml

```diff
@@ -11,7 +11,6 @@ jobs:
       matrix:
         opt: [
           386,
-          enable-acvp-tests,
```

Comment:
These can be enabled and run independently, we should test that.

Reply:
You can enable-acvp-tests independently on enable-fips, but it makes no sense as the test won't be actually executed.